KeyPoint3/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1

```

Comment:
Do not include .settings in pull requests. This is local enviroment specific configuration.